### PR TITLE
add custom airtable api hook

### PR DESCRIPTION
**Pull request by daniaalnahdi**
- added a reusable airtable hook for API calls
- refactored mentors page to use hook

### CODE REVIEW

File: src/components/Mentors.js [hunk 1]
```diff
@@ -18,42 +18,27 @@ const MentorCard = ({ name, position, company, imageURL, active, phone, expertis
   </Col>
 );
 
+export const Mentors = () => {
+  const { data, isLoading } = useAirtableAPI('appexkZgUcQ9vucI9', 'mentors');
```
Comment (feliciazhang) [inline src/components/Mentors.js:22]: maybe it'd be nice to have table ids as constants somewhere? unless theyre only going to be used once

**Review by anikarabenhorst — APPROVED** [review]
lgtm!

File: src/components/Mentors.js (hunk 1 above)
Comment (snwessel) [reply src/components/Mentors.js:22]: I think they should only be used once

File: src/components/Mentors.js [hunk 2]
```diff
@@ -18,42 +18,27 @@ const MentorCard = ({ name, position, company, imageURL, active, phone, expertis
...
+      <Row>
+        <Col>
+          <h1 id='mentors' className='font-weight-bold'>
+            Mentors
+          </h1>
+        </Col>
+      </Row>
+      <Row>
+        {isLoading && <Spinner animation='border' variant='primary' /> /* TODO import custom styled spinner component*/} 
+        {!data.length && <div>Nothing to see here...</div> /* TODO handle empty array in case of API error (or if its actually empty for any reason)*/}
```
Comment (snwessel) [inline src/components/Mentors.js:35]: Thank you for adding this! Now hopefully it won't break the whole page when someone doesn't have the API Key set up yet

**Review by snwessel — APPROVED** [review]
Looks good to me!

**Pull request merged**